labanane/frontend/src/components/playlist.vue

```diff
@@ -13,17 +13,24 @@
         <li v-for='track in playlist.tracks'>{{track.name}}</li>
       </ul>
     </div>
+
+    <youtube-player :ready.sync='ytb_player_ready'></youtube-player>
```

Comment:
`.once` et `.sync` on été [supprimer en v2](lhttps://vuejs.org/v2/guide/migration.html#once-and-sync-Modifiers-on-v-bind-removed) si tu veux remonter une modifier il faut dispatcher un event ou utiliser `vuex`.